macrosynergy/signal/signal_return_relations.py

```diff
@@ -243,9 +244,13 @@ def __init__(
         for sig in self.sigs:
             if self.signs[self.sigs.index(sig)]:
                 self.df.loc[self.df["xcat"] == sig, "value"] *= -1
-                self.df.loc[self.df["xcat"] == sig, "xcat"] = (
-                    self.df.loc[self.df["xcat"] == sig, "xcat"] + "_NEG"
-                )
+                if type(self.df) is QuantamentalDataFrame:
```

Comment:
Should we make this isinstance instead of type so in the future it allows for flexibility for inheritance from QuantamentalDataFrame